documentation/modules/using-must-gather.adoc

```diff
@@ -30,11 +30,38 @@ $ oc adm must-gather --image={must-gather}
 +
 The data is saved as `/must-gather/must-gather.tar.gz`. You can upload this file to a support case on the link:https://access.redhat.com/[Red Hat Customer Portal].
 
-* To gather data for the past 24 hours:
+. Optional: Run the `oc adm must-gather` command with the following options to gather filtered data:
+
+* Namespace:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather} \
+  -- NS=<namespace> /usr/bin/targeted
+----
+
+* Migration plan:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather} \
+  -- PLAN=<migration_plan> /usr/bin/targeted
+----
+
+* Virtual machine:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather} \
+  -- VM=<vm_id> NS=<namespace> /usr/bin/targeted
+----
+
+. Optional: Run the `oc adm must-gather` command with the `gather_metrics_dump` script to gather data for the past 24 hours:
```

Comment:
Same as above, please remove "the past 24 hours".

Reply:
The content in this section is based on the konveyor/forklift-must-gather. The [gather_metrics_dump](https://github.com/konveyor/forklift-must-gather/blob/main/collection-scripts/gather_metrics_dump) script says that it gathers data for 24 hours. Can you confirm that it does not?

Reply:
Ah, sorry, you're right the gather_metrics_dump script is for 24 hours, so it should be kept.   (just the "default" must gather execution doesn't have a time restriction, but also doesn't dump metrics).

Reply:
Got it. metrics_dump collects metrics data for 24 hours but default gather only collects logs and CRs. Will fix.

Reply:
Correct, thank you!